### ISSUE TITLE
PDF viewer stoped working after nextcloud upgrade 10.0.1 > 10.0.2 

### ISSUE THREAD

**Issue by HirIMogul** [nextcloud/server#2614]
### Steps to reproduce
1. Upgrade from 10.0.1 > 10.0.2
2. Open PDF file
3. Get blank page

### Expected behaviour
I open a PDF file and the file is shown with PDF viewer.

### Actual behaviour
I open a PDF file and get a blank page. 

### Server configuration
**Operating system**:
Ubuntu 16.04 LTS
**Web server:**
nginx version: nginx/1.10.0 (Ubuntu)

**Database:**
mysql  Ver 14.14 Distrib 5.7.16, for Linux (x86_64) using  EditLine wrapper

**PHP version:**
php-fpm7.0 -v
PHP 7.0.8-0ubuntu0.16.04.3 (fpm-fcgi)

**Nextcloud version:** (see Nextcloud admin page)
10.0.2 (stable)

**Updated from an older Nextcloud/ownCloud or fresh install:**
updated from 10.0.1 (stable)

**Where did you install Nextcloud from:**
nextcloud.com

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

No errors have been found.

</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder

Enabled:
  - activity: 2.3.2
  - admin_audit: 1.0.0
  - comments: 1.0.0
  - dav: 1.0.1
  - documents: 0.13.1
  - federatedfilesharing: 1.0.1
  - federation: 1.0.1
  - files: 1.5.2
  - files_external: 1.0.2
  - files_pdfviewer: 0.8.1
  - files_retention: 1.0.0
  - files_sharing: 1.0.0
  - files_texteditor: 2.1
  - files_trashbin: 1.0.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 15.0.1
  - notifications: 0.3.0
  - password_policy: 1.0.0
  - provisioning_api: 1.0.0
  - serverinfo: 1.1.1
  - survey_client: 0.1.5
  - systemtags: 1.0.2
  - templateeditor: 0.1
  - theming: 1.0.1
  - updatenotification: 1.0.1
  - workflowengine: 1.0.1
Disabled:
  - calendar
  - encryption
  - external
  - files_accesscontrol
  - files_automatedtagging
  - user_external
  - user_ldap
  - user_saml

</details>

**The content of config/config.php:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)

{
    "system": {
        "updatechecker": false,
        "instanceid": "oc5thjckp7c0",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.10.11",
            "elias-nas.ueperliq0egf9hy3.myfritz.net:4443",
            "cloud.mogul.network"
        ],
        "datadirectory": "\/home\/media2\/owncloud",
        "overwrite.cli.url": "http:\/\/192.168.10.11\/owncloud",
        "dbtype": "mysql",
        "version": "9.1.2.2",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "asset-pipeline.enabled": true,
        "loglevel": 1,
        "maintenance": false,
        "theme": "",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    },
    "apps": {
        "activity": {
            "enabled": "yes",
            "installed_version": "2.3.2",
            "types": "filesystem"
        },
        "admin_audit": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "logging"
        },
        "backgroundjob": {
            "lastjob": "14513"
        },
        "calendar": {
            "enabled": "no",
            "installed_version": "1.4.1",
            "ocsid": "168707",
            "signed": "true",
            "types": ""
        },
        "comments": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "logging"
        },
        "core": {
            "backgroundjobs_mode": "cron",
            "installedat": "1465569081.7643",
            "lastcron": "1481479203",
            "lastupdateResult": "[]",
            "lastupdatedat": "1481479557",
            "oc.integritycheck.checker": "[]",
            "public_documents": "documents\/public.php",
            "public_files": "files_sharing\/public.php",
            "public_webdav": "dav\/appinfo\/v1\/publicwebdav.php",
            "remote_caldav": "dav\/appinfo\/v1\/caldav.php",
            "remote_calendar": "dav\/appinfo\/v1\/caldav.php",
            "remote_carddav": "dav\/appinfo\/v1\/carddav.php",
            "remote_contacts": "dav\/appinfo\/v1\/carddav.php",
            "remote_dav": "dav\/appinfo\/v2\/remote.php",
            "remote_files": "dav\/appinfo\/v1\/webdav.php",
            "remote_webdav": "dav\/appinfo\/v1\/webdav.php",
            "repairlegacystoragesdone": "yes",
            "shareapi_allow_mail_notification": "no",
            "shareapi_allow_public_notification": "no",
            "shareapi_enforce_links_password": "yes",
            "shareapi_only_share_with_group_members": "yes",
            "umgmt_send_email": "true",
            "umgmt_show_backend": "false",
            "umgmt_show_email": "true",
            "umgmt_show_last_login": "true",
            "umgmt_show_storage_location": "false",
            "updater.secret.created": "1481437065",
            "vendor": "nextcloud"
        },
        "dav": {
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": "filesystem"
        },
        "documents": {
            "enabled": "yes",
            "installed_version": "0.13.1",
            "ocsid": "168711",
            "types": ""
        },
        "federatedfilesharing": {
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": ""
        },
        "federation": {
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": "authentication"
        },
        "files": {
            "cronjob_scan_files": "500",
            "default_quota": "100 GB",
            "enabled": "yes",
            "installed_version": "1.5.2",
            "types": "filesystem"
        },
        "files_antivirus": {
            "av_path": "\/usr\/bin\/clamscan",
            "enabled": "no",
            "installed_version": "0.9.0.0",
            "ocsid": "157439",
            "signed": "true",
            "types": "filesystem"
        },
        "files_external": {
            "enabled": "yes",
            "installed_version": "1.0.2",
            "ocsid": "166048",
            "types": "filesystem"
        },
        "files_pdfviewer": {
            "enabled": "yes",
            "installed_version": "0.8.1",
            "ocsid": "166049",
            "types": ""
        },
        "files_retention": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "filesystem"
        },
        "files_sharing": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "filesystem"
        },
        "files_texteditor": {
            "enabled": "yes",
            "installed_version": "2.1",
            "ocsid": "166051",
            "types": ""
        },
        "files_trashbin": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "filesystem"
        },
        "files_versions": {
            "enabled": "yes",
            "installed_version": "1.3.0",
            "types": "filesystem"
        },
        "files_videoplayer": {
            "enabled": "yes",
            "installed_version": "0.9.8",
            "types": ""
        },
        "firstrunwizard": {
            "enabled": "yes",
            "installed_version": "1.1",
            "ocsid": "166055",
            "types": ""
        },
        "gallery": {
            "enabled": "yes",
            "installed_version": "15.0.1",
            "types": ""
        },
        "notifications": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": "logging"
        },
        "password_policy": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "minLength": "5",
            "types": ""
        },
        "provisioning_api": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "prevent_group_restriction"
        },
        "serverinfo": {
            "enabled": "yes",
            "installed_version": "1.1.1",
            "types": ""
        },
        "survey_client": {
            "enabled": "yes",
            "installed_version": "0.1.5",
            "types": ""
        },
        "systemtags": {
            "enabled": "yes",
            "installed_version": "1.0.2",
            "types": "logging"
        },
        "templateeditor": {
            "enabled": "yes",
            "installed_version": "0.1",
            "types": ""
        },
        "theming": {
            "cachebuster": "18",
            "enabled": "yes",
            "installed_version": "1.0.1",
            "name": "Mogul Network",
            "slogan": "a safe place",
            "types": "logging",
            "url": "https:\/\/cloud.mogul.network"
        },
        "updatenotification": {
            "calendar": "1.4.1",
            "core": "9.1.2.2",
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": ""
        },
        "workflowengine": {
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": "filesystem"
        }
    }
}

</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

yes, local

**Are you using encryption:** yes/no

no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

no

### Client configuration
**Browser:**
Chrome  54.0.2840.100 (64-bit)
Firefox 50.0.2

**Operating system:**
Kubuntu 16.04 LTS

### Logs
#### Web server error log
<details>
<summary>Web server error log</summary>
```
2016/12/11 08:51:40 [error] 18799#18799: *744 directory index of "/var/www/" is forbidden, client: 199.181.238.20, server: mogul.network, request: "GET / HTTP$
2016/12/11 13:24:50 [error] 18799#18799: *749 directory index of "/var/www/" is forbidden, client: 82.80.230.228, server: mogul.network, request: "GET / HTTP/$
2016/12/11 14:15:01 [error] 18799#18799: *758 directory index of "/var/www/" is forbidden, client: 199.181.238.20, server: mogul.network, request: "GET / HTTP$
2016/12/11 14:52:36 [error] 18799#18799: *760 directory index of "/var/www/" is forbidden, client: 199.181.238.20, server: mogul.network, request: "GET / HTTP$
2016/12/11 15:21:43 [error] 18799#18799: *762 directory index of "/var/www/" is forbidden, client: 134.249.159.113, server: mogul.network, request: "GET / HTT$
2016/12/11 15:30:04 [error] 18799#18799: *764 directory index of "/var/www/" is forbidden, client: 134.249.159.113, server: mogul.network, request: "GET / HTT$
2016/12/11 15:59:18 [error] 18799#18799: *767 directory index of "/var/www/" is forbidden, client: 184.105.247.195, server: mogul.network, request: "GET / HTT$
2016/12/11 17:09:14 [error] 18799#18799: *836 directory index of "/var/www/" is forbidden, client: 82.80.230.228, server: mogul.network, request: "GET / HTTP/$
2016/12/11 18:39:17 [info] 29826#29826: Using 32768KiB of shared memory for push module in /etc/nginx/nginx.conf:63
2016/12/11 18:50:51 [info] 11522#11522: Using 32768KiB of shared memory for push module in /etc/nginx/nginx.conf:63
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
```
Insert your Nextcloud log here

Does not exist. Increase verbosity?

</details>

#### Browser log
<details>
<summary>Browser log</summary>
...
Insert your browser log here, this could for example include:

a) The javascript console log

/apps/files_pdfviewer/?file=%2Fremote.php%2Fwebdav%2Fdownloads%2FAusweis%2520-%2520Reisepass.pdf:1 Multiple 'X-Frame-Options' headers with conflicting values ('DENY, SAMEORIGIN') encountered when loading 'https://cloud.mogul.network/apps/files_pdfviewer/?file=%2Fremote.php%2Fwebdav%2Fdownloads%2FAusweis%2520-%2520Reisepass.pdf'. Falling back to 'DENY'.
/apps/files_pdfviewer/?file=%2Fremote.php%2Fwebdav%2Fdownloads%2FAusweis%2520-%2520Reisepass.pdf:1 Refused to display 'https://cloud.mogul.network/apps/files_pdfviewer/?file=%2Fremote.php%2Fwebdav%2Fdownloads%2FAusweis%2520-%2520Reisepass.pdf' in a frame because it set 'X-Frame-Options' to 'DENY, SAMEORIGIN'.
jquery.min.js?v=1aaeb59…:2 Uncaught DOMException: Failed to read the 'contentDocument' property from 'HTMLIFrameElement': Blocked a frame with origin "https://cloud.mogul.network" from accessing a cross-origin frame.(…)

</details>

**Comment by LukasReschke** [comment]
>  Multiple 'X-Frame-Options' headers with conflicting values ('DENY, SAMEORIGIN') encountered when loading

You're sending some headers multiple times if you check your HTTP response. That will break stuff. Also DENY is not compatible with Nextcloud.

**Comment by HirIMogul** [comment]
Thank you very much Lukas and sorry for the non-bug report. Your cue tracked me to the right problem. I changed to a let's encrypt certificate. The how to recommended a standard ssl snipet, which was included in my Nginx configuration file for nextcloud. That's why it was doubled. 

I didn't notice this error until I changed to the next version. Again. Sorry for the inconvenience and thank you very much.
